### PR TITLE
Delete all button

### PR DESCRIPTION
**Pull request by GodofCookery**
Delete all queries method added to QueryHistory.js and associated DeleteQueryHistory button added. Deletes all queries on browser in query history but upon page reload previous query still exists.

### CODE REVIEW

File: src/components/QueryHistory.js [hunk 1]
```diff
@@ -144,17 +143,30 @@ export class QueryHistory extends React.Component {
...
+  };
+
+  _deleteQueryHistory = () => {
+    this.historyStore.items = [];
+    this.selectedForTestingStore.items = [];
+
+    console.log('historyStore', this.historyStore);
+    console.log('historyStore.items', this.historyStore.items);
+    console.log('selectedForTestingStore', this.selectedForTestingStore);
+    console.log('selectedForTestingStore.item', this.selectedForTestingStore.items);
```
Comment (JonathanChe) [inline src/components/QueryHistory.js]: Need to remove console logs

File: src/components/QueryHistory.js [hunk 2]
```diff
@@ -187,11 +199,14 @@ export class QueryHistory extends React.Component {
   };
 
   _downloadTestFile = () => {
-    function mochaTest (Q, R) {
+    console.log('historyStore', this.historyStore);
+    console.log('historyStore.items', this.historyStore.items);
+    console.log('selectedForTestingStore', this.selectedForTestingStore);
+    console.log('selectedForTestingStore.item', this.selectedForTestingStore.items);
```
Comment (JonathanChe) [inline src/components/QueryHistory.js]: Need to remove console logs

File: src/components/QueryHistory.js [hunk 3]
```diff
@@ -202,24 +217,29 @@ export class QueryHistory extends React.Component {
...
-    element.href = URL.createObjectURL(file);
-    element.download = "test.txt";
-    element.click();
-  }
+    // arr.push(test);
+    // const element = document.createElement("a");
+    // const file = new Blob(arr, {type: 'text/plain'});
+    // element.href = URL.createObjectURL(file);
+    // element.download = "test.txt";
+    // element.click();
```
Comment (JonathanChe) [inline src/components/QueryHistory.js]: Reason for commenting code out?

**Comment by s-herbert** [comment]
closed for the latest one

**Pull request closed without merging**